python-backend/app/api/nris_services/NRIS_service.py

```diff
@@ -48,13 +57,18 @@ def get_EMPR_data_from_NRIS(mine_no):
     }
 
     headers = {'Authorization': 'Bearer ' + token}
+    try:
+        empr_nris_resp = requests.get(
+            url=current_app.config['NRIS_INSPECTION_URL'],
+            params=params,
+            headers=headers)
+    except requests.exceptions.Timeout:
+        raise
 
-    empr_nris_resp = requests.get(
-        url=current_app.config['NRIS_EMPR_API_URL'],
-        params=params,
-        headers=headers)
-
-    if empr_nris_resp.status_code != 200:
-        return None
+    try:
+        empr_nris_resp.raise_for_status()
+    except requests.exceptions.HTTPError:
+        #TODO add logging for this error.
```

Comment:
Was this just missed or is it out of scope / blocked by something?

Reply:
We currently do not have a way of logging and this was left in to make sure that when we do logging is added here.